.github/workflows/ci.yml

```diff
@@ -130,7 +131,7 @@ jobs:
       - name: Bundle Install
         run: bundle install
       - name: Select Xcode Version
-        run: sudo xcode-select --switch /Applications/Xcode_15.0.1.app/Contents/Developer
```

Comment:
while 15.0.1 is supported by the macOS 14 machines, I've seen a ton of spurious failures on those machines so I'm bumping to 15.2 for better support